src/Workspaces/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexKind.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.RegularExpressions
+{
+    internal enum RegexKind
+    {
+        None,
```

Comment:
Is `None` used? Maybe remove or `None = 0`?

Reply:
i like 'none' as it means we have a way to explicitly check against 'default' tokens and whatnot.

Reply:
Added "= 0" :)

Reply:
I can't find any code referencing RegexKind.None. I didn't check if we have some initialized variables of this type. Would you know if we have some?

---
In reply to: [210060608](https://github.com/dotnet/roslyn/pull/23984#discussion_r210060608) [](ancestors = 210060608)